model/ftn/w3iogomd.ftn

```diff
@@ -3730,7 +3730,7 @@
 !
          DO ITH=1, NTH
 !
-!/OMPG/!$OMP PARALLEL DO PRIVATE(JSEA,ISEA,FACTOR)
+!/OMPG/!$OMP PARALLEL DO PRIVATE(JSEA,ISEA)
 !
             DO JSEA=1, NSEAL
 !/DIST                ISEA         = IAPROC + (JSEA-1)*NAPROC
```

Comment:
These two lines are superfluous

Reply:
> These two lines are superfluous:
> 
> 
@mickaelaccensi this looks like a SBR added by @ardhuin could you please double check if those two lines could be deleted via the link: 
https://github.com/ajhenrique/WW3/blob/456edd6d9bfb01528053cf5f11c2e1c7b9c72057/model/ftn/w3iogomd.ftn#L3736-L3737

Reply:
or it should be more efficient to update the next lines with ISEA instead of JSEA like below:
               ABX(ISEA)  = ABX(ISEA) + A(ITH,IK,ISEA)*ECOS(ITH)
               ABY(ISEA)  = ABY(ISEA) + A(ITH,IK,ISEA)*ESIN(ITH)

Reply:
> or it should be more efficient to update the next lines with ISEA instead of JSEA like below:
> ABX(ISEA) = ABX(ISEA) + A(ITH,IK,ISEA)*ECOS(ITH)
> ABY(ISEA) = ABY(ISEA) + A(ITH,IK,ISEA)*ESIN(ITH)

That makes sense. I will keep things as they are in this PR. @mickaelaccensi can you provide a fix in the Ifremer repo?